global/docker-compose.yml

```
@@ -7,25 +7,24 @@ services:
     cap_add:
       - NET_ADMIN
     command: -A /test/10.0.0.3 --log-facility=-
-    ports:
-        - 127.0.0.1:53:53/udp
-        - 127.0.0.1:53:53
     networks:
       wplocaldocker:
         ipv4_address: 10.0.0.2
   gateway:
-    image: 10up/nginx-proxy
+    image: jwilder/nginx-proxy
```

@dustinrue I think we can use the alpine version, can't we?
```suggestion
    image: jwilder/nginx-proxy:alpine
```